.changeset/strong-ants-think.md

```diff
@@ -0,0 +1,7 @@
+---
+'@lit/reactive-element': patch
+'lit': patch
+'lit-element': patch
+---
+
+Controllers can be added/removed during lifecycle (#4266).
```

Comment:
This may be a breaking change for anyone relying on the broken behavior. I wonder if the changelog should elaborate more on the bug this fixes.

E.g., adding or removing controllers during lifecycle callbacks seems to halt updating other controllers prior to this fix.

Reply:
How about something like:
```suggestion
Fixes bug where adding or removing controllers during a reactive controller lifecycle would affect the execution of other controllers (#4266). Controllers can now be added/removed during lifecycle without affecting others.
```

Reply:
I like it!